src/components/Body/index.jsx

```diff
@@ -1,62 +1,64 @@
-import React from 'react';
-import { Grid, Typography } from '@mui/material';
-import Card from '../Card';
+import Slider from "react-slick";
+import React, { useEffect, useState } from "react";
+import Cards from "../Card";
+import 'slick-carousel/slick/slick.css';
+import 'slick-carousel/slick/slick-theme.css';
+import "./styles.css";
 
-const Body = () => {
+const review = "https://hiddengem.pythonanywhere.com/api/review";
+const users = "https://jsonplaceholder.typicode.com/users";
+
+export default function Body() {
+  const reviewsPerSlide = 3;
+  const [reviews, setReviews] = useState([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
can we move the `fetchData` to src/api/region.js like in this PR please:

```
https://github.com/project-zusammen/hidden-gem-frontend/pull/8
```

try using the env variable as well.

And don't forget to add related tests to your files

Reply:
untuk .env & review.js nya udah saya buat 

tapi saya bingung klo `fetchData` nya di pindahkan ke `src/api/review.js` .
nanti kalo mau nampilkan API di `src/Body/index.jsx`nya gimana mbak?

Reply:
function ny di export di `src/api/review.js` kemudian di import di `src/Body/index.jsx`

bisa lihat contohnya di sini:

https://github.com/project-zusammen/hidden-gem-frontend/pull/8/files

Reply:
uda saya pindahkan mbak, alhamdulillah berhasil.

Reply:
Don't forget to add the unit tests

examples:
https://medium.com/@fatma_2377/a-comprehensive-guide-to-unit-testing-in-react-js-with-vite-ea362d0daade

Reply:
![testing](https://github.com/project-zusammen/hidden-gem-frontend/assets/121326593/c7090bfc-a50d-411e-866a-5a7a2d54da67)

udah baca artikelnya, mbak untuk logic testingnya saya gak paham, 

`import { render } from "@testing-library/react";
import Body from "../components/Body";

test("uses API_URL from .env.test", () => {
  process.env.VITE_API_BASE_URL = "https://hiddengem.pythonanywhere.com/api";
  const { getByText } = render(<Body />);
  //your logic
});`

Reply:
![import meta](https://github.com/project-zusammen/hidden-gem-frontend/assets/121326593/588a7885-a5c5-4666-8423-4ca5f70178c5)

mbak saya udah coba solusinya di internet tapi belum solved juga

Reply:
ini ketika saya melakukan testing